packages/app/src/app/hooks/useInterval.ts

```diff
@@ -0,0 +1,24 @@
+// Based on https://overreacted.io/making-setinterval-declarative-with-react-hooks/
+import { useEffect, useRef } from 'react';
+
+export const useInterval = (callback = () => {}, delay: number) => {
```

Comment:
I think this will be clearer if you extract `noop`

```js
const noop = () => undefined

export const useInterval = (callback = noop, delay: number) => {
```